source/hardware/controllers/reloop_mixage.rst

```diff
@@ -0,0 +1,115 @@
+#       Label                                      Function                                                                                                                                                                     SHIFT function
+======  =========================================  ===========================================================================================================================================================================  ==============================================================================================================================================================================
+1       :hwlabel:`-`                               Allows the temporary deceleration (counter-clockwise) of a track.                                                                                                            Adjust track key down.
+2       :hwlabel:`+`                               Allows the temporary acceleration (clockwise) of a track.                                                                                                                    Adjust track key up.
+3       :hwlabel:`SHIFT`                           When this button is pressed all controls have a different function. The assignment can be viewed in the SHIFT function column.                                               
+4       :hwlabel:`LOOP` / :hwlabel:`IN`            Set beatloop start at current play position and activate beatloop or turn it off if currently active.                                                                        Set loop in point.
+5       :hwlabel:`-LENGTH+` / :hwlabel:`BEATMOVE`  Turn to halve/double loop length; Turn while pushing down to adjust beatjump size; Press shortly to clear any loop points.                                                   Turn to beatjump forward/backward. If a loop is active, move the loop forward/backward; Press shortly to enter/exit beatloop adjustment mode.
+6       :hwlabel:`GAIN`                            Preamplication of a track/deck.                                                                                                                                              Same as non-shift function.
+7       Pitchfader                                 Adjust the pitch or playback speed of the current deck. The LED lights up in the neutral position.                                                                           Same as non-shift function.
+8       :hwlabel:`RELOOP` / :hwlabel:`OUT`         Toggle relooping on/off. Button is lit when a loop is active, blinks slowly if there's an inactive loop set and thus reloop is available.                                    Set loop out point.
+9       :hwlabel:`FX SEL` / :hwlabel:`MASTER`      Select effect in effect rack N for adjustment. Button is lit if an effect slot is selected.                                                                                  Toggle if deck N is sync leader.
```

Comment:
here and below: "effect unit"
```suggestion
9       :hwlabel:`FX SEL` / :hwlabel:`MASTER`      Select effect in effect unit N for adjustment. Button is lit if an effect slot is selected.                                                                                  Toggle if deck N is sync leader.
```
Am I right that units 1 / 2 are assigned by default to deck 1 / 2?

Reply:
Yes.